FirebaseAppDistributionInternal/Sources/ApiService.swift

```diff
@@ -206,7 +206,7 @@ struct CreateFeedbackReportRequest: Codable {
       // we'll have to figure out a way to get the project number
       let urlString = String(
         format: Strings.findReleaseEndpointUrlTemplate,
-        app,
```

Comment:
If we don't want to pass the appId, I think this change should be in the Strings enum?

Reply:
Removing this change. The dash won't actually work. Will fix this in a follow up CL.